module/zfs/vdev_raidz_math_scalar.c

```diff
@@ -142,6 +142,7 @@ static const struct {
 		a.b[6] = mul_lt[a.b[6]];				\
 		a.b[5] = mul_lt[a.b[5]];				\
 		a.b[4] = mul_lt[a.b[4]];				\
+		break;							\
```

Comment:
The fall-thru is intentional here as well.  Is there some way we can annotate this to silence the warnings?